ert3/evaluator/_evaluator.py

```diff
@@ -21,17 +21,14 @@ def _create_evaluator_tmp_dir(workspace_root, evaluation_name):
     )
 
 
-def _assert_single_stage_forward_model(stages_config, ensemble):
-    # The current implementation only support one stage as the forward model
-    # and hence we fail if multiple are provided
-    assert len(ensemble.forward_model.stages) == 1
-
-
-def _prepare_input(ee_config, stages_config, inputs, evaluation_tmp_dir):
+def _prepare_input(ee_config, ensemble, stages_config, inputs, evaluation_tmp_dir):
     tmp_input_folder = evaluation_tmp_dir / "prep_input_files"
-    os.makedirs(tmp_input_folder)
+    os.makedirs(tmp_input_folder, exist_ok=True)
```

Comment:
Is this change really needed?

Reply:
No, just forgot it, it is much easier to test with it running ert3 without the need to always remove the `tmp` folder